easybuild/easyconfigs/d/deap/deap-0.9.2-intel-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PythonPackage'
+
+name = 'deap'
+version = '0.9.2'
+
+homepage = 'http://deap.readthedocs.org/en/master/'
+description = """DEAP is a novel evolutionary computation framework for rapid prototyping and testing of ideas.
+ It seeks to make algorithms explicit and data structures transparent."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+pyver = '2.7.10'
+versionsuffix = '-Python-%s' % pyver
+dependencies = [('Python', pyver)]
+
+pyshortver = '.'.join(pyver.split('.')[:2])
```

Comment:
Actual code in a config. :(
You could simplify this with `pyver[:3]` or `py_maj_min = '2.7'; pyver = '%s.10' % py_maj_min` and then use `py_maj_min` here.